emcc.py

```diff
@@ -1792,6 +1792,21 @@ def default_setting(name, new_default):
 
   setup_environment_settings()
 
+  # Emscripten requires certain ES6 constructs by default in library code
+  # - https://caniuse.com/let              : FF:44 CHROME:49 SAFARI:11
+  # - https://caniuse.com/const            : FF:36 CHROME:49 SAFARI:11
+  # - https://caniuse.com/arrow-functions: : FF:22 CHROME:45 SAFARI:10
+  # - mdn-javascript_builtins_object_assign: FF:34 CHROME:45 SAFARI:9
+  # Taking the highest requirements gives is our minimum:
+  #                             Max Version: FF:44 CHROME:49 SAFARI:11
+  settings.TRANSPILE = (settings.MIN_FIREFOX_VERSION < 44 or
```

Comment:
How about "TRANSPILE_JS"? This entire "compile to the Web" is kind of transpilation too...

Reply:
Its not super important since this is an internal setting, how about: TRANSPILE_TO_ES5

Reply:
Better still.. TARGET_LACK_ES6_FEATURES.